monkey/monkey_island/cc/island_logger.py

```diff
@@ -1,25 +1,45 @@
 import json
 import logging.config
 import os
+from typing import Dict
 
-__author__ = 'Maor.Rayzin'
+from monkey_island.cc.consts import DEFAULT_LOGGER_CONFIG_PATH
 
+__author__ = "Maor.Rayzin"
 
-def json_setup_logging(default_path='logging.json', default_level=logging.INFO, env_key='LOG_CFG'):
+
+def json_setup_logging(
+    default_path=DEFAULT_LOGGER_CONFIG_PATH,
+    default_level=logging.INFO,
+    env_key="LOG_CFG",
+):
     """
     Setup the logging configuration
     :param default_path: the default log configuration file path
     :param default_level: Default level to log from
     :param env_key: SYS ENV key to use for external configuration file path
     :return:
     """
-    path = default_path
+    path = os.path.expanduser(default_path)
     value = os.getenv(env_key, None)
+
     if value:
         path = value
+
     if os.path.exists(path):
-        with open(path, 'rt') as f:
+        with open(path, "rt") as f:
```

Comment:
What's your take on " and '? I usually use `'` for arguments and keys and `"` for strings (in log, exceptions, etc.). You think we should use only one?

Reply:
black picked double quotes in this scenario. There's no recommendation from PEP8 as to which to use in what cases.